elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreTransaction.java

```diff
@@ -104,4 +127,20 @@ Query buildQuery(EntityProjection entityProjection, RequestScope scope) {
     public void cancel() {
         queryEngineTransaction.cancel();
     }
+
+    private void aggregationCompleteQuery(RequestScope scope, QueryResponse response) {
```

Comment:
I think I might try to collapse these three functions into the main logic.  A few reasons:

- Single line functions are often not necessary.
- The isCached logic is tied to code outside the function - so this can't really be reused (the purpose of a function).
- Some of the variables seem like they might be common across the functions.

Reply:
Resolved in latest commit